hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/submerkle/EvmFnResult.java

```diff
@@ -355,6 +366,14 @@ public ContractFunctionResult toGrpc() {
                 grpc.addCreatedContractIDs(createdId.toGrpcContractId());
             }
         }
+        if (!contractNonces.isEmpty()) {
```

Comment:
⚠️ We are not `serialize`-ing and `deserialize`-ing the new `contractNonces` field here.